test/e2e/tests/assets/asset-list.spec.ts

```diff
@@ -118,58 +104,38 @@ describe('Multichain Asset List', function (this: Suite) {
     await withFixtures(
       buildFixtures(this.test?.fullTitle() as string, 137),
       async ({ driver }: { driver: Driver }) => {
-        await loginWithBalanceValidation(driver);
+        await loginWithoutBalanceValidation(driver);
         const headerNavbar = new HeaderNavbar(driver);
         const selectNetworkDialog = new SelectNetwork(driver);
         const assetListPage = new AssetListPage(driver);
+        const sendPage = new SendTokenPage(driver);
         await headerNavbar.clickSwitchNetworkDropDown();
         await selectNetworkDialog.selectNetworkName(NETWORK_NAME_MAINNET);
         await assetListPage.check_tokenItemNumber(4);
         await assetListPage.clickOnAsset('TST');
-        await driver.clickElement('.mm-box > button:nth-of-type(3)');
-        const toastTextElement = await driver.findElement('.toast-text');
-        const toastText = await toastTextElement.getText();
-        assert.equal(
-          toastText,
-          `You're now using ${POLYGON_NAME_MAINNET}`,
-          'Toast text is correct',
-        );
+        await assetListPage.clickSwapButton();
+        await sendPage.check_networkChange(POLYGON_NAME_MAINNET);
       },
     );
   });
   it('shows correct asset and balance when swapping on a different chain', async function () {
     await withFixtures(
       buildFixtures(this.test?.fullTitle() as string),
       async ({ driver }: { driver: Driver }) => {
-        await loginWithBalanceValidation(driver);
+        await loginWithoutBalanceValidation(driver);
         const headerNavbar = new HeaderNavbar(driver);
         const assetListPage = new AssetListPage(driver);
         const selectNetworkDialog = new SelectNetwork(driver);
+        const homePage = new HomePage(driver);
+        const sendPage = new SendTokenPage(driver);
+        const swapPage = new SwapPage(driver);
         await headerNavbar.clickSwitchNetworkDropDown();
         await selectNetworkDialog.selectNetworkName(LINEA_NAME_MAINNET);
         await assetListPage.check_tokenItemNumber(3);
```

Comment:
this step is failing because we don't have 3 assets but 4, as we have Ethereum Mainnet, Linea, Base and Polygon (selected one)

![Screenshot from 2025-06-27 13-26-27](https://github.com/user-attachments/assets/6436788f-9154-4fda-8c77-f8875c8f37b2)